.github/workflows/bump-version.yml

```diff
@@ -44,7 +44,26 @@ jobs:
         re="$(sed 's/[_-]/[_-]/g' <<< "$PACKAGE")"
         sed -i "s/^\($re\)==.*/\1==$VERSION/g" -- requirements*.txt
 
-        sed -i "s/^\($re\) = \".*\"/\1 = \"$VERSION\"/g" -- rust_snuba/Cargo.toml
+        # Update Cargo.toml dependencies (format: package = "version")
+        sed -i "s/^\($re\) = \"[^\"]*\"/\1 = \"$VERSION\"/g" -- rust_snuba/Cargo.toml
+        # Also handle dependencies with features (format: package = { version = "version", features = [...] })
+        sed -i "s/^\($re\) = { version = \"[^\"]*\"/\1 = { version = \"$VERSION\"/g" -- rust_snuba/Cargo.toml
+
+        # Update Cargo.lock if Cargo.toml was modified
+        if ! git diff --exit-code -- rust_snuba/Cargo.toml > /dev/null 2>&1; then
+          cd rust_snuba
+          # Try updating with underscores (cargo prefers underscores in package names)
+          CARGO_PACKAGE="$(echo "$PACKAGE" | sed 's/-/_/g')"
+          if ! cargo update --package "$CARGO_PACKAGE" 2>/dev/null; then
+            # If that fails, try with the original package name
+            if ! cargo update --package "$PACKAGE" 2>/dev/null; then
+              # If both fail, do a full cargo update to be safe
+              echo "Warning: Could not update specific package, running full cargo update"
+              cargo update
```

Comment:
This is guaranteed to update more than the package itself, which is definitely not what we'd want here.

I would not do it and just let a human handle that.